eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -10,4 +10,4 @@ steps:
           exit 1
         }
       displayName: Prevent changes to eng/common outside of azure-sdk-tools repo
-      condition: and(succeeded(), ne(variables['Skip.EngCommonWorkflowEnforcer'], 'true'))
+      condition: and(succeeded(), ne(variables['Skip.EngCommonWorkflowEnforcer'], 'true'), eq('$System.PullRequest.TargetBranch', 'master'))
```

Comment:
I suspect we should do that check in the inline code similar to the source branch check. I also would like to see if there is a good way to get the default branch name instead of hardcoding master.